src/amo/components/Collection/index.js

```diff
@@ -187,7 +183,12 @@ export class CollectionBase extends React.Component<Props> {
           />
         );
       }
-      return <CollectionManager collection={collection} />;
+      return (
+        <CollectionManager
+          collection={collection}
+          page={location.query.page}
```

Comment:
I think we need to import the `withRouter` HOC (and use it) to get a `location` object from it.

Reply:
But we already have a `location` in `this.props`, so why do we need to use the `router` object to get one at this point?

Reply:
The `router` is used to `push` a route update.

Reply:
Oh I see what you mean now. I actually thought we were using `location` from the browser context, not from the routing layer, which is something I did not know about to be honest (the fact that we get a magic `location` injected).

You can ignore my comment on this line then, but if you want to change the `location.query.page` value, the router should be used because what you want to do is changing the URL (and keep things synchronized).